apps/web-app/server/utils/user.ts

```diff
@@ -4,7 +4,7 @@ export async function isUserAdminOfOrg(orgContext: OrgContext) {
   if (!orgContext?.memberId) return false;
   const userOrgMembership = orgContext?.members.find((member) => {
     //@ts-ignore
-    return +member.id === +orgContext?.memberId;
+    return member.id === +orgContext?.memberId;
```

Comment:
```suggestion
    return member.id === orgContext?.memberId;
```
Did you forget the second `+`, or did you intentionally leave it here?

Reply:
Okay, I understand that the + symbol isn't suitable for every ID.

Reply:
yes

Reply:
@muescha is correct, the `orgContext.memberId` is already typed as a number, and so we dont need the `+ `